pom.xml

```
@@ -87,7 +87,7 @@
     <dependency>
       <artifactId>data-model-lib</artifactId>
       <groupId>life.qbic</groupId>
-      <version>2.7.0</version>
+      <version>2.10.0-SNAPSHOT</version>
```

This should be changed to 
```suggestion
      <version>2.10.0</version>
```
as soon as the data-model-lib 2.10.0 is released. 
I'd argue that this blocks this release until DML is released

---

ups, you are right